draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3384,58 +3385,98 @@ dCDN MAY reject such requests for urgent processing using “ereject” error co
+		Given an object list being supported by the dCDN, the CI/T Object List Type capability object is used to 
+		indicate support for one or more Object List types listed in <xref target="IANA.CDNI.ObjectListTypeReg"/> 
+		by the type property of the "ObjectList" object. The capability type is "FCI.CITObjectListType".
         </t>
+        <t><list style="empty" hangIndent="3">
+	<t><list style="hanging" hangIndent="3">
 
-        <t><list style="empty">
-           <t>Property: object-list-types<list style="empty">
+		<t hangText="Name: object-list-types">
+		<vspace blankLines="1"/>
+		Description: A list of supported object list types.
+		<vspace blankLines="1"/>
+		Value: An array <xref target="object-list-type"/>.
+		<vspace blankLines="1"/>
+		Mandatory: Yes.
+		</t>
 
-                 <t>Description: A list of object list types.</t>
+	</list></t>
+	</list></t>
 
-		 <t>Type: A list of Object List Type (from the CDNI CI/T Object List types
-                    <xref target="IANA.CDNI.ObjectListTypeReg"/>)</t>
+        	<section title="CI/T Object List Type Capability Object Serialization">
+		<t>
+             	The following shows an example of a JSON serialized CI/T Object List Type Capability object serialization
+		for a dCDN that supports "hls" and "dash".
+          	</t>
 
-                 <t>Mandatory: No. The default, in case of a missing or an empty list, 
-                    is none supported.</t>
-           </list></t>
-        </list></t>
-        <section title="CI/T Object List Type Capability Object Serialization">
-          <t>
-             The following shows an example of a JSON serialized CI/T Object List Type Capability object serialization
-             for a dCDN that supports "hls" and "dash".
-          </t>
+          	<figure><artwork><![CDATA[
 
-          <figure>
-           <artwork><![CDATA[
+{
+ "capabilities": [
+   {
+     "capability-type": "FCI.CITObjectListType",
+     "capability-value": {
+       "object-list-types": ["hls", "dash", "json"]
+     },
+     "footprints": [ <Footprint objects> ]
+   }
+ ]
+}
+        ]]></artwork></figure>
+        	</section>
+      	</section>
+
+	<section anchor="private-url-capability-object" title="CI/T Private URL Capability Object">
+        <t>
+		The CI/T Private URL capability object is used to indicate support for operations on private URLs 
+		(see <xref target="url-types"/> for details). 
+		The capability type is  "FCI.CITPrivateUrlType".
+        </t>
+        <t><list style="empty" hangIndent="3">
+	<t><list style="hanging" hangIndent="3">
+
+		<t hangText="Name: private-url-type-support">
+		<vspace blankLines="1"/>
+		Description: Indicate whether private URL type is supported by dCDN.
+		<vspace blankLines="1"/>
+		Value: Boolean. 
+		<vspace blankLines="1"/>
+		Mandatory: Yes.
```

Comment:
Is there any reason why an omission of this capability couldn't be interpreted as a lack of support (i.e. defaulting to false)?

Reply:
No, fixed